core/extraction/towns_fund_round_four.py

```diff
@@ -0,0 +1,86 @@
+import pandas as pd
```

Comment:
could we add a docstring at module level, which mentions what Round 4 is? ie that it's Towns Fund data and the reporting dates? We could also add one for the other Reporting round extraction modules?